docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,1995 @@
+{
```

Comment:
<p>Response to "Reason 1":</p><p>Okay, given how simple the functions above are, I think I am okay with you returning ALL discords from <code>_find_discords</code>  and not only the top one since, computationally, there shouldn't be any difference.</p><p> </p><p>Response to "Reason 2":</p><p>Sure, let's check it inside the <code>while</code>  loop and break if we have found all top-K</p><p> </p><p>Of course, if you see my other comment, we won't be appending anymore and so we'd be referencing <code>i</code> (to track where we are in terms of collecting top-K discords) rather than <code>len(discord_idx)</code>     </p><p> </p><p>Let me know if this makes sense</p>

Reply:
> I think I am okay with you returning ALL discords

Yes...computationally there should be probably almost the same. I will change `_find_discords` in next set of commits.



> Sure, let's check it inside the `while` loop and break if we have found all top-K

I will change it in next set of commits.